util/checker/src/cmd/check_stubs_compile.rs

```diff
@@ -0,0 +1,169 @@
+use std::{fs, path::PathBuf, process::Command};
+
+fn make_reserve_copies(modified_exercise: &PathBuf) {
```

Comment:
`PathBuf` and `Path` have the same relationship that `String` and `str` do. It generally makes sense to write a function to take `&Path` unless you need to modify the original, for the same reason that you'd generally take `&str` instead of `&String`.

Reply:
The where some times, when rustc complained that it does not know the size of &Path at compile time, so it cannot be used

Reply:
Are you sure? `Path` isn't sized, but `&Path` should be fine as a function argument.